src/kimp.py

```diff
@@ -1,50 +1,63 @@
 import requests
-import json
+import datetime
 
+#exchange_rate : 6H, upbit_price, binance_price : 1m
 class Kimp:
     upbit_api_url = 'https://api.upbit.com/v1/ticker' #param = markets=
     binance_api_url = 'https://api.binance.com/api/v1/ticker/price' #param = symbol
-    exchange_rate_api_url = ''
-    symbols = ['BTC', 'ETH', 'DOGE']
-    requests_data = requests.get()
+    exchange_rate_api_url = 'https://api.ratesapi.io/api/latest?base=USD&symbols=KRW'
+    requests_data = requests.get(upbit_api_url)
```

Comment:
```suggestion
    requests_data = None
```

명시적으로 값을 나타내는 것이 좋습니다.